### PR TITLE
Actions: Add deprecationMessage metadata syntax field

### PR DESCRIPTION
**Pull request by Simran-B**
### Why:

Document this useful feature for everyone. Discovered on [github.community](https://github.community/t/feature-request-mark-inputs-as-deprecated/171417)

### What's being changed:

- Add description for `deprecationMessage`
- Mark two variables as inline code

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by Simran-B** [comment]
`actions.yml` example:

```yaml
name: Test
description: Action Test
inputs:
  obsolete:
    description: An option to test the deprecationMessage property
    deprecationMessage: This option should no longer be used!
```

How it looks like in the log:

image

**Comment by janiceilene** [comment]
@Simran-B Thanks so much for opening a PR 🙌   It's good to see you around! I'll get this triaged for review 👍

File: content/actions/creating-actions/metadata-syntax-for-github-actions.md [hunk 1]
```diff
@@ -76,6 +76,10 @@ For example, if a workflow defined the numOctocats and octocatEyeColor inputs, t
 
 **Optional** A `string` representing the default value. The default value is used when an input parameter isn't specified in a workflow file.
 
+#### `inputs.<input_id>.deprecationMessage`
+
+**Optional** A `string` to be logged as warning if the input is set. You can notify users to not use this input parameter anymore by setting a deprecation warning for it.
```
Comment (martin389) [inline content/actions/creating-actions/metadata-syntax-for-github-actions.md:81]: Thanks for raising this PR 🎊  I've added a small suggestion here:

```suggestion
**Optional** If the input parameter is used, this `string` is logged as a warning message. You can use this warning to notify users that the input is deprecated and mention any alternatives.
```

**Comment by Simran-B** [comment]
Applied the suggestion!

I think it's fine to not have an example showing use usage of `deprecationMessage` as it's not something you commonly do and it's straightforward anyway.

**Comment by martin389** [comment]
👋 Status update: Checking with dev team

**Review by konradpabjan — APPROVED** [review]
Looks good to me! 👍

**Comment by martin389** [comment]
Looks good in staging: https://docs-4945--patch-1.herokuapp.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddeprecationmessage

**Pull request merged**